### DISCUSSION TITLE
selección de dataset, 5 opciones

### DISCUSSION THREAD

**Discussion by BustamJos3** [rramosp/ai4eng.v1#154] [UDEA Students]
Hola. Quisiera que nos ayudaran a definir cuáles de los siguientes datasets nos pueden servir para el proyecto.
# Datasets
1. [American Express - Default Prediction, predict if a customer will default in the future](https://www.kaggle.com/competitions/amex-default-prediction/data?select=test_data.csv)

Sobre éste en particular, creo que cumple con los 3 1ros requsitos y además lo vemos como un buen candidato, pero el tamaño de los datos es muy grande. Quisiera saber si es posible reducir el tamaño para tener un dataset más manejable (a pesar de que se pueda afectar la predicción), ya sea reduciendo la cantidad de columnas e insertando datos NaN, o de alguna otra forma que nos puedan sugerir.

## Para los demás datasets

El tamaño es más reducido, pero creo que también cumplen los 2 1ros requisitos; en cuanto a los 2 últimos, me parece que se  pueden simular en los casos en que los datasets no los tengan.

2. [Forest Cover Type Prediction, use cartographic variables to classify forest categories](https://www.kaggle.com/competitions/forest-cover-type-prediction/data?select=train.csv)
3. [Predicting Molecular Properties, can you measure the magnetic interactions between a pair of atoms?](https://www.kaggle.com/competitions/champs-scalar-coupling/data?select=mulliken_charges.csv)
4. [Recruit Restaurant Visitor Forecasting, predict how many future visitors a restaurant will receive](https://www.kaggle.com/competitions/recruit-restaurant-visitor-forecasting/data)

@rramosp @raulramosudea @Felipe-RA @andrea-mu

**Comment by rramosp** [comment]
hola bustam, si el dataset es muy grande y te interesa el reto, puedes
simplemente seleccionar un subconjunto de datos que sea manejable y hacer
el proyecto sobre esos

On Thu, Jun 30, 2022 at 5:34 PM BustamJos3 ***@***.***> wrote:

> Hola. Quisiera que nos ayudaran a definir cuáles de los siguientes
> datasets nos pueden servir para el proyecto.
> Datasets
>
>    1. American Express - Default Prediction, predict if a customer will
>    default in the future
>    <https://www.kaggle.com/competitions/amex-default-prediction/data?select=test_data.csv>
>
> Sobre éste en particular, creo que cumple con los 3 1ros requsitos y
> además lo vemos como un buen candidato, pero el tamaño de los datos es muy
> grande. Quisiera saber si es posible reducir el tamaño para tener un
> dataset más manejable (a pesar de que se pueda afectar la predicción), ya
> sea reduciendo la cantidad de columnas e insertando datos NaN, o de alguna
> otra forma que nos puedan sugerir.
> Para los demás datasets
>
> El tamaño es más reducido, pero creo que también cumplen los 2 1ros
> requisitos; en cuanto a los 2 últimos, me parece que se pueden simular en
> los casos en que los datasets no los tengan.
>
>    1. Dog Breed Identification, determine the breed of a dog in an image
>    <https://www.kaggle.com/competitions/dog-breed-identification/data?select=labels.csv>
>    2. Forest Cover Type Prediction, use cartographic variables to
>    classify forest categories
>    <https://www.kaggle.com/competitions/forest-cover-type-prediction/data?select=train.csv>
>    3. Predicting Molecular Properties, can you measure the magnetic
>    interactions between a pair of atoms?
>    <https://www.kaggle.com/competitions/champs-scalar-coupling/data?select=mulliken_charges.csv>
>    4. Recruit Restaurant Visitor Forecasting, predict how many future
>    visitors a restaurant will receive
>    <https://www.kaggle.com/competitions/recruit-restaurant-visitor-forecasting/data>
>
> @rramosp <https://github.com/rramosp> @raulramosudea
> <https://github.com/raulramosudea> @Felipe-RA
> <https://github.com/Felipe-RA> @andrea-mu <https://github.com/andrea-mu>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/154>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIJRAS5QQEWUGH6HHE3VRYOIXANCNFSM52K3MA6A>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
